util/dvsim/dvsim.py

```diff
@@ -99,6 +101,14 @@ def get_proj_root():
     return (proj_root)
 
 
+def handler_ctrl_c(signal_received, frame):
```

Comment:
a little weird to have `signal(SIGINT, ...)` and its handler in different class, WDYT?

Reply:
Just need it in the FlowCfg class.

Reply:
If I move this to `FlowCfg`, the function becomes 3 inputs arguments as the first one is `self`. Need to handle below error. Any thought?
TypeError: sigint_handler() takes 2 positional arguments but 3 were given

Since we call `cfg.kill` function, other children classes should just modify the `kill` function
```
def sigint_handler(signal_received, frame):
    # kill processes and background jobs
    log.debug('SIGINT or CTRL-C detected. Exiting gracefully')
    cfg.kill()
    log.info('Exit due to SIGINT or CTRL-C ')
    exit(1)
```